src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int INDEX_COLOR = 9;
+
+    public String getRandomColor() {
+        Random random = new Random();

```

Comment:
return enum type
move random to field level so you don't create redundant objects

Reply:
what about this comment?

Reply:
???